test/photo-router-test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+require('./mock-env.js');
+
+const expect = require('chai').expect;
+const superagent = require('superagent');
+
+const User = require('../model/user.js');
+const Photo = require('../model/photo.js');
+const Gallery = require('../model/gallery.js');
+
+const userMock = require('./lib/user-mocks.js');
+const galleryMock = require('./lib/gallery-mock.js');
+const serverControl = require('./lib/server-control.js');
+
+const baseURL = `http://localhost:${process.env.PORT}`;
+
+
+describe('testing photo-router', function(){
```

Comment:
**TODO FOR LOST POINTS**

Your before/after block is only running BEFORE any real tests are run because it's enclosed in its own describe block. I get an ECONNREFUSED error which means your tests are unable to run because the server connection already dies.